expyfun/io/_parse.py

```diff
@@ -4,6 +4,39 @@
 
 import numpy as np
 import csv
+import ast
+import json
+from expyfun.stimuli import TrackerUD, TrackerBinom, TrackerDealer
```

Comment:
Somehow I'm not having any luck importing the tracker objects. I tried 
`from expyfun.stimuli import TrackerUD, TrackerBinom, TrackerDealer` and `from ..stimuli import TrackerUD, TrackerBinom, TrackerDealer` but am getting errors for both cases (_ValueError: Attempted relative import in non-package_ and _ImportError: cannot import name TrackerUD_ respectively). Any thoughts?